pages/data_life_cycle/sharing.md

```diff
@@ -81,10 +60,23 @@ Based on the considerations listed above, you should be able to determine the ri
 - **Controlled access or Data Access Committees (DACs)**: data can only be shared with researchers, whose research is reviewed and approved by a Data Access Committee (DAC). DAC is an organization of one or more named individuals responsible for data release to external requestors based on specific criteria (research topics, allowed geographical regions, allowed recipients etc). Criteria established by DAC for data access are usually described on the website of the organization.
 - **Access upon request (not recommended)**: in order to manage this type of access a named contact is required for the dataset who would be responsible for making decisions about whether access is granted. The owner of the data must provide his/her contact in the documentation associated with the datasets (metadata). Metadata about the datasets must be open.
 
-Share and publish your data in professional deposition databases that provide the appropriate access type and licence.
+  <!-- ERIM specific info (begin)-->
+
+## What options do I have for sharing my data and materials as an ERIM member? 
+
+For open access article publishing, EUR has done a lot of deals with the most common publishers, so please check if your journal is already part of these deals, you can look for your journal in the [EUR Journal Browser](https://www.google.com/url?sa=t&rct=j&q=&esrc=s&source=web&cd=&cad=rja&uact=8&ved=2ahUKEwjP2pr0o4mAAxWzXaQEHWQiBM8QFnoECBkQAQ&url=https%3A%2F%2Fwww.eur.nl%2Fen%2Flibrary%2Fresearch-support%2Fopen-access%2Fopen-access-eur%2Fpublisher-deals&usg=AOvVaw2DGgTBttoYteaEj6BEWjUd&opi=89978449)
```

Comment:
The link here included Google's URL. I updated it so that it will go directly to the EUR website.  👍 
```suggestion
For open access article publishing, EUR has done a lot of deals with the most common publishers, so please check if your journal is already part of these deals, you can look for your journal in the [EUR Journal Browser](https://www.eur.nl/en/library/research-support/open-access/open-access-eur/publisher-deals)
```